src/bin/vip-cache-purge-url.js

```diff
@@ -64,8 +70,8 @@ command( {
 	appQuery,
 	envContext: true,
 	wildcardCommand: true,
-	usage: 'vip cache purge-url <URL>',
+	usage,
 } )
-	.option( 'from-file', 'Read URLs from file (useful to purge multiple URLs)' )
+	.option( 'from-file', 'Read URLs from a file, can be used to purge multiple URLs.' )
```

Comment:
I see that you provided more instructions for the use of  `--from-file` in the command example, but I'm wondering if we could include that in the description instead. If a user is reading this in the help menu for `vip cache purge-url`, they already have the context for using `--from-file` for purging a URL from cache. So maybe the description could simply be:
> Read one or more URLs from a file, each listed on a single line.

Reply:
Good call! I pushed an update to use that description.